scripts/checkout_web_vault.sh

```diff
@@ -34,7 +34,7 @@ if [ ! -d "${VAULT_FOLDER}" ]; then
     pushd "${VAULT_FOLDER}"
         # If this is the first time, init the repo and checkout the requested branch/tag/hash
         git -c init.defaultBranch=main init
-        git remote add origin https://github.com/bitwarden/clients.git
+        git remote add origin https://github.com/vaultwarden/vw_web_builds.git
```

Comment:
I'd rename it vaultwarden so it does not interfere with already checked out repositories

```suggestion
        git remote add vaultwarden https://github.com/vaultwarden/vw_web_builds.git
```
(This would then have to be changed on line 58 too to fetch from vaultwarden.)